code/ui/blocks/src/controls/Number.tsx

```diff
@@ -64,6 +64,10 @@ export const NumberControl: FC<NumberProps> = ({
     if (inputValue !== newInputValue) {
       setInputValue(value);
     }
+    console.log('useEffect', value, newInputValue, inputValue);
```

Comment:
@chakAs3 Can you remove the `console.log()` here?

Reply:
@cdedreuille yes sure